actors/common/src/main/java/org/sunbird/models/user/User.java

```diff
@@ -68,6 +68,7 @@
   private List<String> locationIds;
   private String prevUsedPhone;
   private String prevUsedEmail;
+  private Boolean isStateValidated;
```

Comment:
instead of object we can use wrapper boolean .

Reply:
made changes to all boolean values